x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -232,62 +235,60 @@ const excludeLinks = (linkIds: SecurityPageName[]) => ({
   links: links.links?.filter((link) => !linkIds.includes(link.id)),
 });
 
-const getHostIsolationExceptionTotal = async (http: CoreStart['http']) => {
-  const hostIsolationExceptionsApiClientInstance =
-    HostIsolationExceptionsApiClient.getInstance(http);
-  const summaryResponse = await hostIsolationExceptionsApiClientInstance.summary();
-  return summaryResponse.total;
-};
-
 export const getManagementFilteredLinks = async (
   core: CoreStart,
   plugins: StartPlugins
 ): Promise<LinkItem> => {
   const fleetAuthz = plugins.fleet?.authz;
-  const { endpointRbacV1Enabled } = ExperimentalFeaturesService.get();
-  const hasPermissionsForSecuritySolution = calculatePermissionsFromCapabilities(
-    core.application.capabilities
-  );
+
+  const { endpointRbacEnabled, endpointRbacV1Enabled } = ExperimentalFeaturesService.get();
+  const isEndpointRbacEnabled = endpointRbacEnabled || endpointRbacV1Enabled;
```

Comment:
Shouldn't we just use the `endpointRbacEnabled` flag for `8.7` and remove the use of `v1`?

Reply:
Yeah, probably... I left it in just in case these changes make it to 8.6 via a backport. I'm hoping we can delete these FF soon.